### PR TITLE
Makes it easier to get an implanter and debugger

### PR DESCRIPTION
**Pull request by SonicGotNuked**
Third times the charm

Implanter - <s>is hidden under hacked autolathe equipment and corresponding material value was created.</s> Found it to be better to add them as a research requirement for the protolathe, will keep its material value.

Debugger - <s>Added in autolathe regards to construction overhaul providing future use of these.</s> per skulls orders, will work on them to be orderable via cargo.

**NOTE**
This is NOT the circuit debugger for IE, this is infact the one that spawns inside electrical lockers everyone looks at and never uses.

### CODE REVIEW

**Comment by SonicGotNuked** [comment]
!wip

File: html/changelogs/sonicgotnuked-autolathe.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: SonicGotNuked
+
+delete-after: true
+
+changes:
+  - rscadd: "Adds the implanter to protolathe with corsponding material value and research."
+  - rscadd: "Adds debugger to cargo for 100 credits"
```
Comment (Arrow768) [inline html/changelogs/sonicgotnuked-autolathe.yml]: That changelog line is incorrect.
What you did is make central pay you 100 credits for a shipped debugger (/datum/export)

**Comment by SonicGotNuked** [comment]
!wip

**Comment by SonicGotNuked** [comment]
!review

**Comment by SonicGotNuked** [comment]
https://github.com/Aurorastation/Aurora.3/pull/6070/files made this PR outdated.

Closing

**Pull request closed without merging**